### PR TITLE
Template stand

### PR DESCRIPTION
**Pull request by mary3000**
Yet another stand: now with custom carbon-clickhouse ([fork](https://github.com/mary3000/carbon-clickhouse)) that supports templating (`old_graphite` section in config), and with grafana's template variables.
[produced image](https://imgur.com/XOwNLkX)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,7 @@
...
-stand/data
-stand/snapshots
-stand/grafana/key.txt
\ No newline at end of file
+
+.idea
+
+*/data
+*/snapshots
+key.txt
```
Comment (code-of-kpp) [inline .gitignore]: add a newline please

File: template-stand/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - "./init.sql:/init.sql"
+  entrypoint: |
+    bash -c '
+      sleep 5
+      export HOME=/var/lib/clickhouse/
+      cat /init.sql | clickhouse client --host clickhouse --multiquery'
+  links:
+  - clickhouse
+carbon-clickhouse:
+  image: sham42/carbon-clickhouse:latest
```
Comment (code-of-kpp) [inline template-stand/docker-compose.yaml:21]: could you please provide a link here?

**Review by code-of-kpp — CHANGES_REQUESTED** [review]
add a newlines please and mention this new demo in README

File: template-stand/docker-compose.yaml (hunk 2 above)
Comment (mary3000) [reply template-stand/docker-compose.yaml:21]: https://github.com/mary3000/carbon-clickhouse

File: .gitignore (hunk 1 above)
Comment (mary3000) [reply .gitignore]: fixed

**Comment by code-of-kpp** [comment]
could you please rebase and mention your carbon-clickhouse fork and corresonding PR in readme?

**Comment by code-of-kpp** [comment]
https://github.com/lomik/carbon-clickhouse/pull/32

**Comment by mary3000** [comment]
rebased

**Review by code-of-kpp — APPROVED** [review]

**Pull request merged**